### ISSUE TITLE
Difference with jupynium?

### ISSUE THREAD

**Issue by Ciel-MC** [SUSTech-data/neopyter#3]
Im not very familiar with notebooks but I can’t help but notice the similarities between this and https://github.com/kiyoon/jupynium.nvim could you perhaps explain how it is different? I am planning on using jupyter as a part of my workflow trying out some ML, so details would be helpful, thanks

**Comment by AbaoFromCUG** [comment]
I need to introduce `JupyterLab ` and `JupyterNotebook` firstly, according https://jupyter.org/

> JupyterLab: A Next-Generation Notebook Interface
JupyterLab is the latest web-based interactive development environment for notebooks, code, and data. Its flexible interface allows users to configure and arrange workflows in data science, scientific computing, computational journalism, and machine learning. A modular design invites extensions to expand and enrich functionality.

> Jupyter Notebook: The Classic Notebook Interface
The Jupyter Notebook is the original web application for creating and sharing computational documents. It offers a simple, streamlined, document-centric experience.

Here are some different between neopyter and jupynium.nvim
||Neopyter|Jupynium.nvim|
|--|--|--|
|Feature|preview/run in `jupyter lab`|preview/run in `jupyter notebook`|
|Limitation|Jupyter lab only|Jupyter notebook only|
|Architecture|`Nvim plugin`+`JupyterLab Extension`|`Nvim plugin`+`Selenium`|

In summary, Neopyter is an alternative of jupynium.nvim if you use `JupyterLab`

In addition, https://github.com/kiyoon/jupynium.nvim/issues/74 is a discuss about jupynium&Jupyter Notebook 7
